src/vivarium_public_health/risks/distributions.py

```diff
@@ -84,12 +105,22 @@ def ppf(self, x):
         category_index = (exposure_sum.T < x).T.sum('columns')
         return pd.Series(np.array(self.categories)[category_index], name=self.risk + '_exposure', index=x.index)
 
+    def __str__(self):
+        return f"PolytomousDistribution(risk= {self.risk}, categories= {self.categories}"
+
+    def __repr__(self):
```

Comment:
These reprs for the distributions seem a little inconsistent - sometimes they include e.g. exposure_data, means, sd, etc. just as strings, sometimes they include the data, sometimes they aren't included at all.